examples/websocket-client-basic-auth/websocket_client_basic_auth.bal

```diff
@@ -0,0 +1,22 @@
+import ballerina/io;
+import ballerina/websocket;
+
+// Defines the WebSocket client to call the Basic auth secured APIs.
+// The client is enriched with the `Authorization: Basic <token>` header by
+// passing the `websocket:CredentialsConfig` for the `auth` configuration of the
+// client.
+websocket:Client securedEP = check new("wss://localhost:9090/foo/bar",
+    auth = {
+        username: "ldclakmal",
+        password: "ldclakmal@123"
+    },
+    secureSocket = {
+        cert: "../resource/path/to/public.crt"
+    }
+);
+
+public function main() returns error? {
+    check securedEP->writeTextMessage("Hello, World!");
+    string response = check securedEP->readTextMessage();
```

Comment:
```suggestion
    string response = check securedEP->readTextMessage();
```
When we say the `response` isn't it more like getting a response for a request as in HTTP. Shall we keep it as  `textMessage`? WDYT?

Reply:
+1. Fixed with https://github.com/ballerina-platform/ballerina-distribution/pull/1930/commits/845ed035766a43de7340176f08ef9799c6df422f